pkgs/by-name/ap/apkeditor/update.rb

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i ruby -p nix-prefetch-git
```

Comment:
```suggestion
#!nix-shell -i ruby -p nix-prefetch-git ruby
```
you still need the Ruby package to provide a `ruby` executable to you